package.json

```diff
@@ -77,7 +77,6 @@
     "chalk": "4.1.1",
     "chokidar": "^3.4.2",
     "cookie": "^0.4.2",
-    "graphql": "^16.3.0",
```

Comment:
We should still keep `graphql` as a dev dependency so we could develop the library. I suppose newer versions of NPM install peer dependencies out of the box, but let's keep it explicit.